### ISSUE TITLE
Setting Error foreground or background colors is ignored.

### ISSUE THREAD

**Issue by jdhitsolutions** [PowerShell/PSReadLine#554]
Environment data
----------------
PS version: 5.1.16275.1000
PSReadline version: 1.2
os: 10.0.16275.1000 (WinBuild.160101.0800)
PS file version: 10.0.16275.1000 (WinBuild.160101.0800)

Steps to reproduce or exception report
--------------------------------------
Run Set-PSReadlineOption and change any combination of error foreground or background color.
```
Set-PSReadlineOption  -ErrorForegroundColor white -ErrorBackgroundColor red
```
I can see the change with Get-PSReadlineoption, but when I type a command with an error, the colors are ignored. I get the default red foreground. Is this now hard-coded?

**Comment by lzybkr** [comment]
I believe this is fixed in #561.

I'll publish a pre-release version to the PowerShell Gallery as soon as they support pre-releases.

If you'd like to try a fix before that, you can build the lastest yourself from the master branch, or 
grab a build from AppVeyor like this [one](https://ci.appveyor.com/api/buildjobs/ust6h6k216i5gm81/artifacts/bin%2FRelease%2FPSReadline.zip).

Note the version number is now 2.0, so be sure to copy the contents to <your module dir>\PSReadLine\2.0.

**Comment by lzybkr** [comment]
I'll have an official prerelease build out next week if you want to verify this is fixed, otherwise you can grab a build out of appveyor [here](https://ci.appveyor.com/project/lzybkr/psreadline).
